lib/mllp/receiver.ex

```diff
@@ -41,7 +41,8 @@ defmodule MLLP.Receiver do
           dispatcher: module(),
           packet_framer: module(),
           transport_opts: :ranch.opts(),
-          context: map()
+          context: map(),
+          ref: reference()
```

Comment:
This should be `term()` here since we pass it on to ranch's child spec, which takes a ref argument of any. It's a bit confusing because in the code we pass this in via `args.receiver_id`, which defaults to `make_ref()` if no `ref` arg is given.